c++/examples/CMakeTests.cmake

```diff
@@ -16,17 +16,31 @@
 ##############################################################################
 ##############################################################################
   # Remove any output file left over from previous test run
+set (CPP_EX_CLEANFILES
+    Group.h5
+    SDS.h5
+    SDScompound.h5
+    SDSextendible.h5
+    Select.h5
+)
 add_test (
     NAME CPP_ex-clear-objects
     COMMAND    ${CMAKE_COMMAND}
-        -E remove
-             Group.h5
-             SDS.h5
-             SDScompound.h5
-             SDSextendible.h5
-             Select.h5
+        -E remove ${CPP_EX_CLEANFILES}
+)
+set_tests_properties (CPP_ex-clear-objects PROPERTIES
+    FIXTURES_SETUP clear_cppex
+    WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
```

Comment:
These two are identical right? Can we not just set the FIXTURES_SETUP and FIXTURES_CLEANUP properties on the same test and not have duplicates everywhere?

Reply:
If I read the CMake documentation correctly, FIXTURES_SETUP and FIXTURES_CLEANUP cannot be the same test. Therefore the dups. Actually the SETUP could probably be deleted since the clean should have cleanedthe files.